jerry-core/profiles/README.md

```diff
@@ -37,3 +46,78 @@ This command selects the profiles/minimal.profile file.
 Set FEATURE_PROFILE option to one of the following values:
 * the profile with absolute path
 * name of the profile (which needs to exist in the `profiles` folder)
+
+
+### Configurations
+
+In JerryScript all of the features are enabled by default, so an empty profile file turns on all of the available ECMA features.
+
+---
+#### CONFIG_DISABLE_BUILTINS
```

Comment:
This part looks like hard to maintain.

Reply:
Maybe, but it is needed. I think these won't change too often.